ReleaseNotes.md

```diff
@@ -7,6 +7,3 @@ This is a list of publicly available software that is redistributed, categorized
 
 ### MIT ###
 * [Foundation v5](http://foundation.zurb.com)
-
-### Other licenses ###
```

Comment:
we might want to leave this in. While we aren't pulling in normalize separately, we are still using it directly.